src/execution/index/art/node256.cpp

```diff
@@ -47,6 +47,16 @@ idx_t Node256::GetNextPos(idx_t pos) {
 	return Node::GetNextPos(pos);
 }
 
+idx_t Node256::GetNextPosAndByte(idx_t pos, uint8_t &byte) {
+	for (pos == DConstants::INVALID_INDEX ? pos = 0 : pos++; pos < 256; pos++) {
```

Comment:
Maybe a nitpick: can we use a `do while` here, or split this up somehow?
I think it's being too clever and it's a little hard to follow

Reply:
I have a separate PR after this one to replace all the magic numbers (this is not the only time in the ART code haha), it is part of the track-index-size refactor.

Reply:
If that works for you, I'd like to leave it like this for now, and I'll add these changes to my other branch avoiding merge conflicts for all the `nodeXX.cpp` files?